tools/text_processing_deployment/sh_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
How about moving this file to the `tests/`?

Reply:
> 2021 -> 2023 NVIDIA CORPORATION -> NVIDIA CORPORATION & AFFILIATES

Done 

> How about moving this file to the `tests/`?

Uses `export_grammars.sh` and planning to merge this script back into `export_grammars.sh` in the future so that testing would remain consistent